### DISCUSSION TITLE
.NET MAUI updates in .NET 9 Preview 3

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#21783] [Announcements]
The team is continuing work on core fundamentals of the .NET MAUI SDK to improve overall product quality. This includes expanding test coverage, end to end scenario testing, and bug fixing. 

Here's a summary of what's new in .NET MAUI in this preview release:

- [Multi-target versions of .NET for iOS bindings](https://github.com/dotnet/core/blob/main/release-notes/9.0/preview/preview3/dotnetmaui.md#net-for-ios)
- [Android Asset Packs](https://github.com/dotnet/core/blob/main/release-notes/9.0/preview/preview3/dotnetmaui.md#asset-packs)

.NET MAUI updates in .NET 9 Preview 3:
* [Release notes](https://github.com/dotnet/core/blob/main/release-notes/9.0/preview/preview3/dotnetmaui.md) 
* [GitHub Release](https://github.com/dotnet/maui/releases/tag/9.0.0-preview.3.10457)
* [How to install](https://github.com/dotnet/core/blob/main/release-notes/9.0/install-workloads.md)

.NET 9 Preview 3:
* [Discussion](https://aka.ms/dotnet/9/preview3)
* [Release notes](https://github.com/dotnet/core/blob/main/release-notes/9.0/preview/preview3/README.md)

**Comment by jeromelaban** [comment]
@davidortinau I'm having trouble installing the workloads on windows (it works for preview 2). Using a blank machine (in Windows Sandbox), I'm installing .NET 9 preview 3 then running:

```
C:\Users\WDAGUtilityAccount>dotnet workload install maccatalyst ios android maui

Installing Microsoft.MacCatalyst.Sdk.net9.0_17.2.Msi.x64 ..... Done
Downloading Microsoft.MacCatalyst.Sdk.net8.0_17.0.Msi.x64 (17.0.8519)
Workload installation failed. Rolling back installed packs...
Removing Microsoft.MacCatalyst.Sdk.net9.0_17.2.Msi.x64 ..... Done
Workload installation failed: One or more errors occurred. (Version 17.0.8519 of package microsoft.maccatalyst.sdk.net8.0_17.0.msi.x64 is not found in NuGet feeds https://api.nuget.org/v3/index.json.)
```

Is there a custom feed that I need to add for Preview 3?

Thanks!

**Reply by rolfbjarne** [reply]
> Is there a custom feed that I need to add for Preview 3?

I believe adding this feed will work:

> https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json

See also https://github.com/dotnet/sdk/issues/40221.

**Reply by rolfbjarne** [reply]
Should be fixed now.

**Reply by jeromelaban** [reply]
It's installing fine now, thanks!

**Comment by rolfbjarne** [comment]
> Is there a custom feed that I need to add for Preview 3?

I believe adding this feed will work:

> https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json

See also https://github.com/dotnet/sdk/issues/40221.

**Comment by rogerbriggen** [comment]
Hi @davidortinau,
I don't know where to file bugs for the preview. There is a regression with this ticket number https://github.com/dotnet/maui/issues/22318 which was introduced with preview 3.
Hope this helps.
